modules/ossm-operator-install.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-operator-install_{context}"]
+= Installing the operator
+
+The {ProductShortName} installation process introduces an operator to manage the installation of the control plane within the `istio-operator` namespace. This operator defines and monitors a custom resource related to the deployment, update, and deletion of the control plane.
+
+.Prerequisites
+
+* Review the link:https://github.com/Maistra/istio-operator/tree/maistra-0.12/deploy/examples[operator templates on GitHub].
```

Comment:
No GitHub links without clearance from support.

Reply:
@knrc and @brian-avery Please review Kathryn's comment on GitHub links. Do we have clearance for support to link to GitHub here?

Reply:
This link installs the operator during Tech Preview while we are working on the OLM integration, it will not be required for GA.

Reply:
I am leaving this link here with the knowledge that we are still in TP and it will be removed for GA.